<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_Calibrate">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>710</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>701</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="widget_RobotSensor" native="true">
    <widget class="QTableWidget" name="tableWidget_Sensor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>884</width>
       <height>341</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-15</y>
       <width>781</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#QGroupBox{padding-top:15px; margin-top:-20px}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="2,5,0,2,5,1">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_RobotType">
        <item>
         <property name="text">
          <string>UR3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Universal Robotics 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_RobotIp">
        <property name="text">
         <string>IP Address</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_RobotType">
        <property name="text">
         <string>Robot Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_PackagePath"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_Plate">
        <item>
         <property name="text">
          <string>RTR_UR3_CHESSBOARD_1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RTR_UR3_CHESSBOARD_2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Plate">
        <property name="text">
         <string>Plate Selection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_PackagePath">
        <property name="text">
         <string>Package Path</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="button_PacakgePath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QLineEdit" name="lineEdit_RobotIp"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_Connect">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>89</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>901</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_ThreeDView" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_CameraView" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
